easybuild/easyconfigs/k/KrakenUniq/KrakenUniq-1.0.3-foss-2022a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'Binary'
+
+name = 'KrakenUniq'
+version = '1.0.3'
+
+homepage = 'https://github.com/fbreitwieser/krakenuniq/'
+description = """KrakenUniq: confident and fast metagenomics classification using unique k-mer counts"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+github_account = 'fbreitwieser'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['559b45081688118b4872e1b9153f83d0ae3414649a1b299ab4f9996d9e154403']
+
+dependencies = [
```

Comment:
Seeing the error on Generoso, I think it needs Perl as a dependency (it seems to be missing a particular Perl package when running the `sanity_check_commands`)